src/gameComponents/AttackPool.java

```diff
@@ -1,4 +1,32 @@
 package gameComponents;
 
 public class AttackPool {
+    private boolean spentBrace = false;
+    private boolean spentEvade = false;
+    private boolean spentContain = false;
+    private boolean spentScatter = false;
+    private boolean spentRedirect = false;
```

Comment:
don't forget salvo. Also thoughts for / against having an arraylist of spentTokens / tokentypes, and just seeing if future tokens are members of that list

Reply:
Yeah, that's a better solution than a list of bools.

Reply:
Opted to strip this out entirely.  I'd prefer that this object really be about holding the current dice.  No need to have defense tokens mucking this up.  This block probably has a better home in the proposed Attack object that is serving as the go between for all things related to attacks, in this instance the go between for ships/tokens and the attack pool.